draft-zip.md

```diff
@@ -0,0 +1,107 @@
+
+# Motivation
+
+While ZIP-XXX (Establishing the Zcash Sustainability Fund) describes a method by which funds can be added to the Zcash Sustainability Fund by a voluntary `ZSF_DEPOSIT` transaction field. The default value of this field is zero and it is left up to the app, wallet, and mining software implementers to make use of it.
+
+This ZIP takes a much more explicit and non-optional approach, mandating at the protocol level that 60% of transaction fees be deposited into the ZSF. As noted above, implementing this change allows the ZSF to accrue value earlier and contribute to future network sustainability.
+
+If ZIPs ### and ### are accepted, the system looks something like this:
+
+At Every New Block:
+- `ZSF_DEPOSIT` amount is deposited into the ZSF
+- Block rewards come from the ZSF
+- Transaction fees (inputs - outputs) paid to miner
+
+After the features described in this ZIP are activated (changed parts in bold):
+
+At Every New Block:
+- `ZSF_DEPOSIT` amount is deposited into the ZSF
+- Block rewards come from the ZSF
+- **40% of transaction fees (inputs - outputs) paid to miner**
+- **60% of transaction fees (inputs - outputs) deposited into the ZSF**
+
+This has a multitude of benefits:
+
+1. **Network Sustainability**: This mechanism involves temporarily reducing the supply of ZEC similar to asset burning in Ethereum’s EIP-1559, but with potential long-term sustainability benefits as the redistribution of deposits contributes to issuance rewards and network development, making it an attractive option for current and future Zcash users.
+2. **Ecosystem Benefits of Longer Time Horizons**: A reliable and long-term functioning Zcash blockchain allows users to make secure long-term plans, leading to a sustainable and virtuous adoption cycle, rather than being influenced by short-term trends.
+3. **Incentivizing Transaction Inclusion**: By providing a 40% share of transaction fees to miners, this ZIP maintains incentives for miners to prioritize including transactions in their blocks. This helps ensure the efficient processing of transactions and supports a robust and responsive network.
+4. **Future-Proofing the Network**: Diverting transaction fees into the ZSF is a forward-looking approach that prepares the Zcash network for future challenges and opportunities. It establishes a financial buffer that can be instrumental in addressing unforeseen issues and seizing strategic advantages as the Zcash ecosystem evolves.
+
+# Specification
+
+This ZIP only proposes a single modification to the transaction fees:
+1. Keep the current destination of 40% of the fees untouched, but route 60% of the fees back to the ZSF
+
+## Transaction fee routing requirements
+
+- For each transaction, 60% of the total fee MUST be paid to the ZSF
+- Any fractions are rounded in favour of the miner
```

Comment:
This is a requirement on the specification, rather than specification itself. (If it were the latter, then it would be redundant with line 78.) Therefore it should not use "MUST".